components/UnorderedList/src/index.scss

```diff
@@ -1,28 +1,9 @@
 @import "~@utrecht/components/unordered-list/css";
 
-.denhaag-unordered-list {
-  @extend .utrecht-unordered-list;
+.denhaag-unordered-list--nested {
+  @extend .utrecht-unordered-list--nested;
```

Comment:
Gaat dit dan straks stuk als we css moeten gaan importeren?

Reply:
Geen idee! We importeren en extenden op meerdere plaatsen utrecht css, dus laten we bijv. morgen even kijken naar wat we allemaal moeten wijzigen voor die wijziging aan de kant van Utrecht.